crates/storage/Cargo.toml

```
@@ -16,3 +16,4 @@ prost      = { workspace = true }
 serde      = { workspace = true }
 
 [dev-dependencies]
+test-case  = { workspace = true }
```

Remove an extra space

```suggestion
test-case = { workspace = true }
```